File: src/main/java/com/fox/automation/page/DefaultPage.java
package com.fox.automation.page;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;

public class DefaultPage {
    WebDriver driver;

    public DefaultPage(WebDriver driver) {
        this.driver = driver;
    }

    public void click_AccountLink() {
        String xpath = "//a[@href='/account']";
//        WebDriverWait wait = new WebDriverWait(driver, 10);
//        wait.until(
//                ExpectedConditions.and(
//                        ExpectedConditions.presenceOfElementLocated(By.xpath(xpath)),
//                        ExpectedConditions.visibilityOfElementLocated(By.xpath(xpath))
//                )
//        );
        WebElement webElement = driver.findElement(By.xpath(xpath));
        webElement.click();
    }
}

File: python-selenium/pages/defaultpage.py
# -*- coding: utf-8 -*-

from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC

class DefaultPage:
    def __init__(self, webdriver):
        self.webdriver = webdriver

    def click_AccountLink(self):
        xpath = "//a[@href='/account']"
        element = self.webdriver.find_element_by_xpath(xpath)
        element.click()

